Source/Xamarin/Prism.Forms/Navigation/Xaml/NavigateToExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Xamarin.Forms;
+
+namespace Prism.Navigation.Xaml
+{
+    [ContentProperty(nameof(Name))]
+    public class NavigateToExtension : Navigation
+    {
+        public bool Animated { get; set; } = true;
+        public string Name { get; set; }
+        public bool? UseModalNavigation { get; set; } = null;
```

Comment:
I'm thinking the UseModalNavigation & Animated properties need to go as we've deprecated their use in favor of the querystring... @brianlagunas thoughts?

Reply:
Yeah, I would leave them off for now. We can always add them later if necessary. It's always harder to remove something than it is to add later.

Reply:
Done